src/intorods/filesys/fs_smb.py

```diff
@@ -152,7 +152,7 @@ def lsfilenames(self, path):
             result.append(entry.filename)
         return result
 
-    def mkdir(self, path):
+    def mkdir(self, path, parents=False):
         self.smb_conn.createDirectory(self.share, uxtowin(path))
```

Comment:
Can we make this adhere to the **parents** parameter?

Reply:
should be simple, I copied the implementation of the irods fs. It uses the folderexists method of the respective fs class. Note: this method uses `self.exists(path)` for the smb fs, but this exists method does not seem to be implemented.